sovtokenfees/sovtokenfees/main.py

```diff
@@ -155,7 +154,11 @@ def register_three_pc_hooks(node, token_ledger, token_state, fees_tracker):
                                       three_pc_handler.check_recvd_pre_prepare)
 
 
-def register_trackers():
+def register_trackers(node, token_state, token_ledger):
     # TODO: move trackers into write_manager
     fees_tracker = BatchFeesTracker()
```

Comment:
Why don't we put `fees_tracker` to `db_manager` as well?
With this change we can avoid throwing it throwing it through all these builders.